From 058c2bf47deeab574dd41353b0d87132327ece8b Mon Sep 17 00:00:00 2001
From: Ashish Sawakar <asawakar@qti.qualcomm.com>
Date: Fri, 27 Mar 2026 11:57:39 +0530
Subject: [PATCH 107/223] ANDROID: ABI: Add to QCOM symbols list

1 function symbol(s) added

arch_invalidate_pmem

Bug: 498119843
Bug: 506790426
Change-Id: I0bb46979418db0b53cfdd77a74c99115abb1b6bf
Signed-off-by: Ashish Sawakar <asawakar@qti.qualcomm.com>
Signed-off-by: Madhu Ananthula <mananthu@qti.qualcomm.com>
(cherry picked from commit edb0a7fb0f6f0a621521131da19866f460d95dc5)
---
 gki/aarch64/abi.stg      | 10 ++++++++++
 gki/aarch64/symbols/qcom |  1 +
 2 files changed, 11 insertions(+)

diff --git a/gki/aarch64/abi.stg b/gki/aarch64/abi.stg
index 7c0cee7157fc..326b7296061c 100644
--- a/gki/aarch64/abi.stg
+++ b/gki/aarch64/abi.stg
@@ -457393,6 +457393,15 @@ elf_symbol {
   type_id: 0x33756485
   full_name: "arch_freq_scale"
 }
+elf_symbol {
+  id: 0x97ee5cc4
+  name: "arch_invalidate_pmem"
+  is_defined: true
+  symbol_type: FUNCTION
+  crc: 0x645584e6
+  type_id: 0x1567dd7e
+  full_name: "arch_invalidate_pmem"
+}
 elf_symbol {
   id: 0xb515aa54
   name: "arch_pick_mmap_layout"
@@ -529516,6 +529525,7 @@ interface {
   symbol_id: 0x520c1997
   symbol_id: 0x2d3715da
   symbol_id: 0x7ff45ec2
+  symbol_id: 0x97ee5cc4
   symbol_id: 0xb515aa54
   symbol_id: 0x0c805860
   symbol_id: 0xc447be59
diff --git a/gki/aarch64/symbols/qcom b/gki/aarch64/symbols/qcom
index 35146af7cff2..89b36e3a1e57 100644
--- a/gki/aarch64/symbols/qcom
+++ b/gki/aarch64/symbols/qcom
@@ -43,6 +43,7 @@
   __arch_copy_from_user
   __arch_copy_to_user
   arch_freq_scale
+  arch_invalidate_pmem
   arch_stack_walk
   arch_timer_read_counter
   argv_free
-- 
2.50.1

